rampwf/kits/ramp_kit.py

```diff
@@ -9,10 +9,10 @@
 BASE_RAMP_KIT_URL = 'https://github.com/ramp-kits/'
 
 RAMP_KITS_AVAILABLE = ('kaggle_seguro',
-                       'mars_craters',
+                       # 'mars_craters',
                        'MNIST_simplified',
                        'MNIST',
-                       'california_rainfall_test',
+                       # 'california_rainfall_test',
```

Comment:
`ramp-workflow` has now been removed from the dependency list in `mars_craters` kit